galaxy/main/migrations/0108_sanitize_namespace_names.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-05-31 13:18
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+STRIP_VERSION_SUFFIX = """
```

Comment:
Nit picky, I know, but this variable name should be related to the task at hand. Look like it was copy-pasta'd from some other migration.

Reply:
+1